pkg/services/cloudmigration/api/api.go

```diff
@@ -51,6 +51,15 @@ func (cma *CloudMigrationAPI) registerEndpoints() {
 	}, middleware.ReqGrafanaAdmin)
 }
 
+// swagger:route POST /cloudmigration/token migrations
```

Comment:
need to define an operation id
```suggestion
// swagger:route POST /cloudmigration/token migrations  createCloudMigrationToken
```